ledger/acctupdates.go

```diff
@@ -281,19 +278,14 @@ func (au *accountUpdates) loadFromDisk(l ledgerForTracker, lastBalancesRound bas
 
 // close closes the accountUpdates, waiting for all the child go-routine to complete
 func (au *accountUpdates) close() {
-	if au.voters != nil {
-		au.voters.close()
+	if au.accountsq != nil {
+		au.accountsq.close()
```

Comment:
huh, so we never used to close these prepared statements?

Reply:
well, yes :)